other-docs/guides/launching-a-site-on-altis.md

```diff
@@ -0,0 +1,126 @@
+
+### Example launch checklist
+
+```markdown
+# Goal
+
+<!-- A brief overview of the launch objective. Is it a migration? Is it a new site? -->
+
+## Pre-launch checklist
+<!-- To be filled out by the project team/lead. Not needed for Altis Support ticket. -->
+- [ ] Are we deploying the correct branch?
+- [ ] Has AWS SES been setup?
+- [ ] Is a content freeze active?
+- [ ] Is the client ready for DNS switches?
+- [ ] Site is ready for HTTPS?
+- [ ] Are any redirections needed?
+
+## Domains hosted on Altis
+<!-- List of domains that will be on the stack -->
+
+### Production
+- domain.com
+- domain2.com
+- etc.
+
+### Staging
+- domain-staging.altis.cloud
+- domain2-staging.altis.cloud
+- etc.
+
+### Development
+<!-- If applicable -->
+- domain-dev.altis.cloud
+- domain2-dev.altis.cloud
+- etc.
+
+**What domain(s) will email need to be sent from?**
+<!-- Enter as a list -->
+- noreply@domain.com
+- etc.
+
+**For each site in the network, what will be the FROM email?
+<!-- Enter as a list -->
+- **Network Admin:** noreply@domain.com
+- **domain.com:** some-email@domain.com
+- etc.
+
+## Environment
+* **GitHub repository:** https://github.com/{owner}/{repo}
+* **GitHub repository deploy branches:**
+  * **Production:** `main`
+  * **Staging:** `staging`
+  * **Development:** `dev`
+* **Database import link:**
+* **Uploads import link:**
+* **Who is providing the SSL certificate(s) (Altis or client)?:** 
+* **Who controls the DNS for the client (Altis or client)?:**
+* **Projected traffic:** 
+* **Any other specifications or challenges?**
+_Refer to the Altis [Limitations](https://docs.altis-dxp.com/cloud/limitations/) and [Page Caching](https://docs.altis-dxp.com/cloud/page-caching/) guides for specifics around what requests can be supported._
+
+
+```
+
+## Preparing Altis Cloud Environments
+After you have submitted your support request in the Altis Dashboard, the Altis Support team will work on setting up your environments and be in touch with your team. When your new cloud environments are set up, you will be contacted and can begin deploying your code to those new environments.
+
+Once you have environments set up, you should restrict access to them by [requiring login](https://docs.altis-dxp.com/security/require-login/), forcing [PHP basic authentication](https://docs.altis-dxp.com/security/php-basic-auth/), or both. In your Altis configuration file (`composer.json`), add the following to require login both and PHP authentication. You may omit PHP authentication, but we recommend having at least one of these enabled for all non-production and pre-launch sites. You can override the require login setting to not require login on local environments. PHP authentication is disabled by default on local environments. Refer to the documentation pages for more information about Require Login and PHP Basic Authentication.
+
+```json
+"altis": {
```

Comment:
I know it gets deeply nested but I'm trying to keep the example JSON configs complete so the full `{ "extra": { "altis" : { ... } } }`, at the very least this should be valid JSON so the code highlighting doesn't get confused.